### DISCUSSION TITLE
Is tractography available?

### DISCUSSION THREAD

**Discussion by ebrahimebrahim** [now-i-know-my-abcd/docs#68] [Where in the World is…? (Help)] [answered]
Are tractograms available?

I see in the data dictionary that there are tables containing measures from AtlasTrack processing of the dMRI data.
But I am wondering: are the actual tracts themselves available? And if so then how can I access them?

**Comment by laikevin128** [comment, marked as answer]
You can find them at NITRC https://www.nitrc.org/ under AtlasTrack, thanks

**Reply by ebrahimebrahim** [reply]
I see, so the idea to download AtlasTrack and register to it and then pull back the tracts provided by AtlasTrack into subject space. Thanks!
